feedback/src/webapp/js/feedback.js

```diff
@@ -82,7 +83,7 @@
                     $('#feedback-technical-setup-instruction').show();
                 }
 
-                if (feedback.supplementaryInfo.length > 0) {
+                if (feedback.supplementaryInfo.length == 0 && feedback.mailSupport==null) {
```

Comment:
Yes, the logic before said (incorrectly) to show the message box (saying that you need to set a property) if the property was set - which was not right.  

Now it says show the message box if neither of 2 properties is set ('feedback.supplementaryInfo, mail.support) (if you set either then you get some functionality).